mongodb_exporter.go

```diff
@@ -42,47 +41,42 @@ func defaultMongoDBURI() string {
 }
 
 var (
-	versionF       = flag.Bool("version", false, "Print version information and exit.")
-	listenAddressF = flag.String("web.listen-address", ":9216", "Address to listen on for web interface and telemetry.")
-	metricsPathF   = flag.String("web.metrics-path", "/metrics", "Path under which to expose metrics.")
-
-	collectDatabaseF   = flag.Bool("collect.database", false, "Enable collection of Database metrics")
-	collectCollectionF = flag.Bool("collect.collection", false, "Enable collection of Collection metrics")
-	collectTopF        = flag.Bool("collect.topmetrics", false, "Enable collection of table top metrics")
-	collectIndexUsageF = flag.Bool("collect.indexusage", false, "Enable collection of per index usage stats")
-
-	uriF     = flag.String("mongodb.uri", defaultMongoDBURI(), "MongoDB URI, format: [mongodb://][user:pass@]host1[:port1][,host2[:port2],...][/database][?options]")
-	tlsF     = flag.Bool("mongodb.tls", false, "Enable tls connection with mongo server")
-	tlsCertF = flag.String("mongodb.tls-cert", "", "Path to PEM file that contains the certificate (and optionally also the decrypted private key in PEM format).\n"+
+	versionF       = kingpin.Flag("version", "Print version information and exit.").Short('v').Bool()
+	listenAddressF = kingpin.Flag("web.listen-address", "Address to listen on for web interface and telemetry.").Default(":9216").String()
+	metricsPathF   = kingpin.Flag("web.metrics-path", "Path under which to expose metrics.").Default("/metrics").String()
```

Comment:
Canonical name is `web.telemetry-path`: https://github.com/prometheus/node_exporter/blob/f028b816152f6d5650ca2cd707e45cda7333fdc1/node_exporter.go#L138